src/shims-vtk.d.ts

```diff
@@ -295,3 +246,49 @@ declare module '@kitware/vtk.js/Widgets/Widgets3D/ImageCroppingWidget' {
   };
   export default vtkImageCroppingWidget;
 }
+
+// declare module '@kitware/vtk.js/Widgets/Widgets3D/ResliceCursorViewWidget' {
+declare module '@kitware/vtk.js/Widgets/Widgets3D/ResliceCursorWidget' {
+  import vtkAbstractWidget from '@kitware/vtk.js/Widgets/Core/AbstractWidget';
+  import vtkWidgetState from '@kitware/vtk.js/Widgets/Core/WidgetState';
+  import type { Vector3 } from '@kitware/vtk.js/types';
+  import vtkResliceCursorWidget from '@kitware/vtk.js/Widgets/Widgets3D/ResliceCursorWidget';
+
+  export interface vtkResliceCursorViewWidget extends vtkAbstractWidget {
+    setManipulator(manipulator: vtkPlaneManipulator): boolean;
+    getManipulator(): vtkPlaneManipulator;
+    setImage(image: vtkImageData): void;
+    setKeepOrthogonality(keepOrthogonality: boolean): void;
+    setScaleInPixels(scaleInPixels: boolean): void;
+    setCursorStyles(styles: {
+      translateCenter: string,
+      rotateLine: string,
+      translateAxis: string,
+    }): boolean;
+  }
+
+  export interface ResliceCursorWidgetState extends vtkWidgetState {
+    placeWidget(bounds: Bounds): void;
+    setScale1(scale: number): boolean;
+    setScale3(scaleX: number, scaleY: number, scaleZ: number): boolean;
+    getPlanes(): Record<number, { normal: Vector3, viewUp: Vector3, color3: Vector3 }>;
+    setPlanes(planes: Record<number, { normal: Vector3, viewUp: Vector3, color3: Vector3 }>): void;
+    getCenter(): Vector3;
+    setScrollingMethod(mode: number): boolean;
+    setOpacity(opacity: number): boolean;
+  }
+
+  // Just forwarding vtk-js's definition as default export:
+  export default vtkResliceCursorWidget;
+}
+
+declare module '@kitware/vtk.js/Proxy/Representations/GeometryRepresentationProxy' {
+  // import vtkGeometryRepresentationProxy from '@kitware/vtk.js/Proxy/Representations/GeometryRepresentationProxy';
+  import vtkAbstractRepresentationProxy from '@kitware/vtk.js/Proxy/Core/AbstractRepresentationProxy';
+  export interface vtkGeometryRepresentationProxy extends vtkAbstractRepresentationProxy {
+    setVisibility(visible: boolean): boolean;
+    getVisibility(): boolean;
+  }
+
+  export default vtkGeometryRepresentationProxy;
+}
```

Comment:
it would be better to directly expose those in VTK.js

Reply:
Typescript definitions added here: https://github.com/Kitware/vtk-js/pull/2906